packages/manager/src/utilities/formatStorageUnits.ts

```
@@ -0,0 +1,14 @@
+const storageRegex = /([0-9])([kMGTPEZY]?i?[B])/;
+
+function replaceFunc(match: string, p1: string, p2: string) {
+  if (match.length === 0) return 'unknown';
```

I think a better/safer alternative here is the string itself. I believe defaulting to "Unknown" its actually not necessary anymore, since we are requesting even shadow plan types in AuthenticationWrapper (with `ensureAllTypes()`).

---

I guess it should actually throw an error or return null or something else. This is supposed to be a generic function that doesn't only work on plan names but instead that works where ever we want to ensure consistent spacing with storage units.

---

But if it's run on a string without storage units? E.g. if we introduce a new plan name without storage units in the title, we'd wouldn't want this to throw an error or return null... we'd just want the original string.

---

Good point!